docs/date.md

```diff
@@ -1,96 +1,398 @@
+
+**Returns:** a new `Temporal.Date` object (or the same object if `thing` was a `Temporal.Date` object.)
+
+This static method creates a new `Temporal.Date` object from another value.
+If the value is a string, it must be in ISO 8601 format.
+If the string designates a time zone, the time zone will be ignored.
+If the value is another `Temporal.Date` object, the same object is returned.
+If the value is any other object, it must have `year`, `month`, and `day` properties, and a `Temporal.Date` will be constructed from them.
+
+Note that any time or time zone part of an ISO 8601 string passed to this function is optional, and will be ignored.
+
+Example usage:
+```javascript
+date = Temporal.Date.from('2006-08-24');  // => 2006-08-24
+date = Temporal.Date.from('2006-08-24T15:43:27');  // => 2006-08-24
+date = Temporal.Date.from('2006-08-24T15:43:27Z');  // => 2006-08-24
+date = Temporal.Date.from('2006-08-24T15:43:27+01:00[Europe/Brussels]');
+  // => 2006-08-24
+date === Temporal.Date.from(date)  // => true
+
+date = Temporal.Date.from({year: 2006, month: 8, day: 24});  // => 2006-08-24
+date = Temporal.Date.from(Temporal.DateTime.from('2006-08-24T15:43:27'));
+  // => same as above; Temporal.DateTime has year, month, and day properties
+```
 
-Allows for easier comparison of `Date` objects, returns:
+### Temporal.Date.**compare**(_one_: Temporal.Date, _two_: Temporal.Date) : number
 
-- `-1` if the first object represents a lower value than the second
-- `0` if the 2 objects represent the same value
-- `1` if the first object represents a higher value than the second
+**Parameters:**
+- `one` (`Temporal.Date`): First date to compare.
+- `two` (`Temporal.Date`): Second date to compare.
+
+**Returns:** &minus;1, 0, or 1.
+
+Compares two `Temporal.Date` objects.
+Returns an integer indicating whether `one` comes before or after or is equal to `two`.
+- &minus;1 if `one` comes before `two`;
+- 0 if `one` and `two` are the same;
+- 1 if `one` comes after `two`.
+
+This function can be used to sort arrays of `Temporal.Date` objects.
+For example:
+```javascript
+one = Temporal.Date.from('2006-08-24');
+two = Temporal.Date.from('2015-07-14');
+three = Temporal.Date.from('1930-02-18');
+sorted = [one, two, three].sort(Temporal.Date.compare);
+sorted.join(' ');  // => 1930-02-18 2006-08-24 2015-07-14
+```
 
 ## Properties
 
 ### date.**year** : number
 
-Returns the year this `Date` represents
-
 ### date.**month** : number
 
-Returns the month this `Date` represents
-
 ### date.**day** : number
 
-Returns the day this `Date` represents
+The above read-only properties allow accessing each component of the date individually.
+
+Usage examples:
+```javascript
+date = Temporal.Date.from('2006-08-24');
+date.year   // => 2006
+date.month  // => 8
+date.day    // => 24
+```
 
 ### date.**dayOfWeek** : number
 
-Returns the day of the week this `Date` represents
+The `dayOfWeek` read-only property gives the weekday number that the date falls on.
```

Comment:
**Here and elsewhere:** In general, you should say that these properties are calendar-specific.  Something like:

> The `dayOfWeek` getter returns the weekday number that the date falls on, calling the corresponding function on the `calendar` field.

Reply:
(later)